sdks/agent-sdk/src/utils/message.ts

```diff
@@ -15,6 +15,12 @@ type ContentMessage = Pick<
   "content" | "contentType" | "fallback"
 >;
 
+export const hasDefinedContent = <ContentTypes = unknown>(
+  message: DecodedMessage<ContentTypes>,
+): message is DecodedMessage<ContentTypes> & {
+  content: NonNullable<ContentTypes>;
+} => !!message.content;
```

Comment:
prefer `message.content === undefined` here since some current or future content types may include types that cast to `false`

Reply:
Ok. Macroscope also wants me to check against `null`.

Reply:
Yes but there is the `NonNullable` type which hints at `null` values.

Reply:
probably best to drop `NonNullable` and just use `undefined`